packages/eslint-plugin/src/lib/configs/all.ts

```diff
@@ -2,7 +2,7 @@ import type { TSESLint } from '@typescript-eslint/utils';
 import rules from '../rules';
 
 const commonConfig: TSESLint.FlatConfig.Config = {
-  files: ['*.ts, *.js'],
```

Comment:
Unbelieveable: I have checked that line before but didn't see the issue.
Following my tests, the line must be like that:
```js
files: ['**/*.ts', '**/*.js'],

Reply:
Oh yes, you're right. I will update my PR :)